src/metabase/api/search.clj

```diff
@@ -216,6 +216,17 @@
   {:arglists '([model search-context])}
   (fn [model _] model))
 
+(defn collection-id-for-table
```

Comment:
Do we need to use this in `add-model-index-permissions-clause` as well? It uses `model.collection_id` to enforce perms.

Reply:
Yes we do, thanks for pointing this out!